src/main/java/weizberg/earthquake/EarthquakeFrame.java

```diff
@@ -50,23 +50,38 @@ public void actionPerformed(ActionEvent actionEvent) {
                         .observeOn(SwingSchedulers.edt())
                         //.observeOn(AndroidSchedulers.mainThread()) // Instead use this on Android only
                         .subscribe(
-                                (response) -> handleResponse(response),
+                                (response) -> {
+                                    handleResponse(response);
+                                    String[] listData = new String[featureCollection.features.length];
+                                    for (int i = 0; i < featureCollection.features.length; i++) {
+                                        Feature feature = featureCollection.features[i];
+                                        listData[i] = feature.properties.mag + " " + feature.properties.place;
+                                    }
+                                    jlist.setListData(listData);
```

Comment:
Instead of duplicating this code in two places, put it back in `handleResponse()`